app.py

```diff
@@ -18,20 +18,23 @@
             "FQDN": "prod.schematic.io",
             "CERTIFICATE_ARN": "arn:aws:acm:us-east-1:878654265857:certificate/d11fba3c-1957-48ba-9be0-8b1f460ee970",
             "TAGS": {"CostCenter": "NO PROGRAM / 000000"},
+            "SCHEMATIC_CONTAINER_LOCATION": "ghcr.io/sage-bionetworks/schematic:v0.1.94-beta",
         }
     case "stage":
         environment_variables = {
             "VPC_CIDR": "10.254.193.0/24",
             "FQDN": "stage.schematic.io",
             "CERTIFICATE_ARN": "arn:aws:acm:us-east-1:878654265857:certificate/d11fba3c-1957-48ba-9be0-8b1f460ee970",
             "TAGS": {"CostCenter": "NO PROGRAM / 000000"},
+            "SCHEMATIC_CONTAINER_LOCATION": "ghcr.io/sage-bionetworks/schematic:v0.1.94-beta",
```

Comment:
We will be updating this in a future PR with the image we want to deploy out to that environment. I'm leaving these as the latest beta image for now, but we'll have a new one shortly.

Reply:
I'm not opposed to this change however I'm wondering about the motivation?    I'm not a fan because it makes it easy for users to disregard the promotion deployment workflow described in the README.

Reply:
Personally, I have always been a fan of using Trunk-Based development over long-lived feature branches per environment. This change supports that idea. A write-up on the two: https://ardalis.com/trunk-based-development-vs-long-lived-feature-branches/

For this specific change, my thought process is this:

1. These changes complement the flow proposed in the readme. Specifying the image in this way means each branch is going to get a copy of this configuration and it's controlled in a single spot in the code for each environment. Compared to being controlled in the specific long-lived branch. The issue comes into play when long-lived branches end up diverging from eachother in small ways.
2. As a developer I can specify the container to be used in each environment and not prevent promoting other changes that might need to go ahead of an image update. Suppose I am making a change to the network stack, container resources, or something else - Unrelated to bumping up the schematic image - That I am able to release without also bringing an image update with it (Or messing around with reverting the image for the dev branch, promoting changes, then un-reverting the dev branch).

Reply:
I think these argument assumes that the container images are de-coupled from the infra environment.  I believe that the containers plus the infra that they run in should be treated as one cohesive unit.  Thinking that the same version of a set of containers will work the same in a different infra environment is risky for deployments.  With that being said I'm still not opposed if you want to work this way.

Reply:
I appreciate the perspective! I agree, that most of the time infra and applications should move as a cohesive unit so we can verify them hand-in-hand. But I also want to support the flexibility in cases where they aren't moving in lockstep.